### PR TITLE
see features in use by domain (+ prefactoring)

### PR DESCRIPTION
**Pull request by nickpell**
Ops needs to provide a list of domains and get a table containing which priority paid features are in use by the domain, so they know what is the minimum plan that a client can be put on without interrupting features in use.

This PR prefactors the code used to alert users when a downgrade will result in used features being lost, and then adds the management command to generate the needed table.

### CODE REVIEW

File: corehq/apps/accounting/subscription_changes.py [hunk 1]
```diff
@@ -67,85 +58,75 @@ def get_response(self):
...
-    @property
-    def response_inbound_sms(self):
+    @staticmethod
+    def response_inbound_sms(domain):
         """
         All Reminder rules utilizing "survey" will be deactivated.
         """
         try:
-            surveys = filter(lambda x: x.method in [METHOD_IVR_SURVEY, METHOD_SMS_SURVEY], self._active_reminders)
+            surveys = filter(lambda x: x.method in [METHOD_IVR_SURVEY, METHOD_SMS_SURVEY], _active_reminders(domain))
```
Comment (dimagimon) [inline corehq/apps/accounting/subscription_changes.py]: E501 line too long (117 > 115 characters)

**Comment by nickpell** [comment]
@benrudolph @biyeun cc: @dannyroberts

File: corehq/apps/accounting/subscription_changes.py [hunk 2]
```diff
@@ -67,85 +58,78 @@ def get_response(self):
         return len(filter(lambda x: not x, response)) == 0
 
 
+# TODO - cache
```
Comment (benrudolph) [inline corehq/apps/accounting/subscription_changes.py]: is this something you were going to do in this PR?

**Comment by benrudolph** [comment]
looks good to me and nothing blocking

File: corehq/apps/accounting/subscription_changes.py (hunk 2 above)
Comment (nickpell) [inline corehq/apps/accounting/subscription_changes.py]: I'm going to briefly punt this and come back to it soon

**Pull request merged**